src/Command/I18nExtractCommand.php

```diff
@@ -31,6 +31,14 @@
  */
 class I18nExtractCommand extends Command
 {
+    /**
+     * @inheritDoc
```

Comment:
It should be `{@inheritDoc}` everywhere

Reply:
we  used both syntax in cake but now `stickler-ci ` is unhappy with `@inheritDoc'
I can change it to see what happen

Reply:
`{@inheritDoc}` for inline heriting. It's supposed to be `@inheritDoc` to inherit the whole docblock.

Our 3.x usage inaccurate which has been fixed in 4.x.